specification/hybridnetwork/resource-manager/Microsoft.HybridNetwork/preview/2022-01-01-preview/operation.json

```diff
@@ -0,0 +1,133 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "HybridNetworkManagementClient",
+    "description": "The resources in this swagger specification will be used to manage the operations.",
+    "version": "2022-01-01-preview"
```

Comment:
>     [](http://example.com/codeflow?start=0&length=4)

Consider using the common-type for this : [https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v3/types.json#L358](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v3/types.json#L358)

Reply:
Thank you for the suggestion. Since this time we only add two APIs on top of the existing stable API version for public preview, for definitions other than the two APIs do you mind if we just keep the same to the stable API version at this moment?

Reply:
Changed it to refer to the one defined in the common-type.